<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>922</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: #333333;
border-radius: 2px;
}
QLabel {
	color: #fffff;
}
QToolButton {
border: 1px solid #B5B5B5;
border-radius: 12px;
color: white;
padding-left: 4px;
padding-right: 4px;
}
QToolButton::hover {
border: 1px solid white;
color: white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="create_torrent_form_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="form_header" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_31">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="form_title">
          <property name="styleSheet">
           <string notr="true">padding: 12px 12px 8px 12px;
font-size: 16px;
font-weight: bold;
color: white;</string>
          </property>
          <property name="text">
           <string>Create a new torrent</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_cancel">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 4px;
padding: 4px;
margin-right:8px;
border: None;
background-color:#555;</string>
          </property>
          <property name="text">
           <string>CANCEL (X)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="form_body" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QFormLayout" name="formLayout_6">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="1">
         <widget class="QLineEdit" name="create_torrent_name_field">
          <property name="styleSheet">
           <string notr="true">color:#000;
border:1px solid #555;
background-color:#555;
padding:4px;</string>
          </property>
          <property name="placeholderText">
           <string>Torrent name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTextEdit" name="create_torrent_description_field">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#000;border: 1px solid #555;
background-color:#555;
padding:4px;</string>
          </property>
          <property name="placeholderText">
           <string>Torrent description</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QListWidget" name="create_torrent_files_list">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>120</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #303030;
border: 1px solid #555;
color: #000;
background-color:#555;
padding:4px;</string>
          </property>
          <item>
           <property name="text">
            <string>/a/b/c/d/e</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/d/e/f/g/h</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="seed_after_adding_checkbox">
          <property name="styleSheet">
           <string notr="true">color: #bbb;</string>
          </property>
          <property name="text">
           <string>Seed this torrent after adding it</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QWidget" name="file_chooser" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_33">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QToolButton" name="create_torrent_choose_dir_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>26</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius: 13px;
padding-left: 4px;
padding-right: 4px;</string>
             </property>
             <property name="text">
              <string>Choose Diretory</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="hspace_fs_1">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="create_torrent_choose_files_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16666666</width>
               <height>26</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius: 13px;
padding-left: 4px;
padding-right: 4px;</string>
             </property>
             <property name="text">
              <string>Choose Files</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="hspace_fs_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">margin-top:8px; color: #fff;</string>
          </property>
          <property name="text">
           <string>Torrent files</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QCheckBox" name="add_to_channel_checkbox">
          <property name="styleSheet">
           <string notr="true">color: #bbb;</string>
          </property>
          <property name="text">
           <string>Add this torrent to My Channel</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QWidget" name="file_selector" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_33">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="file_export_dir">
             <property name="styleSheet">
              <string notr="true">color:#000;
  border:1px solid #555;
  background-color:#555;
  padding:4px;</string>
             </property>
             <property name="text">
              <string>/home/tribler</string>
             </property>
             <property name="placeholderText">
              <string>Save to directory</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="save_directory_chooser">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>26</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius: 13px;
padding-left: 4px;
padding-right: 4px;
margin-left: 8px;</string>
             </property>
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="form_footer" native="true">
       <property name="styleSheet">
        <string notr="true">background-color:#333;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_32">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <item>
         <spacer name="hspace_footer_1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>570</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="edit_channel_create_torrent_progress_label">
          <property name="styleSheet">
           <string notr="true">color:#fff;</string>
          </property>
          <property name="text">
           <string>Creating torrent. Please wait...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_create">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777212</width>
            <height>24</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 12px;
padding-left: 4px;
padding-right: 4px;</string>
          </property>
          <property name="text">
           <string>CREATE TORRENT</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
